components/KidProfile/index.js

```diff
@@ -19,11 +21,11 @@ export default function KidProfile({
         <h2>{kidData.name} </h2>
         <h3>* {formatDate(kidData.birthDate)}</h3>  
       <StyledEventList>
-      {kidData?.events?.map((event) => (
-          <li key={event._id}>
-           <p>{event.title}</p>
-           <p>{formatDate(event.date)}</p>
-          </li>))}
+        {kidData?.events?.map((event) => (
+          <Event
+          key={event._id} event={event} kidData={kidData} mutate={mutate}
```

Comment:
The prop `kidData` is named differently here and in the `Event` component.
Decide which name to use, either `kitData` or `kidsData`.